Assets/Scripts/Utils/ProfileManager.cs

```diff
@@ -80,7 +80,8 @@ static string GetProfile()
             var hashedBytes = new MD5CryptoServiceProvider()
                 .ComputeHash(Encoding.UTF8.GetBytes(Application.dataPath));
             Array.Resize(ref hashedBytes, 16);
-            return new Guid(hashedBytes).ToString("N");
+            // Authentication service only allows profile names of maximum 30 characters
```

Comment:
Should we also include something in `UIProfileSelector` to  make sure we don't go over the 30 characters cap? Maybe clamping the string in `UIProfileSelector.SanitizeProfileName`?